src/components/component.ts

```diff
@@ -67,6 +70,38 @@ module Plottable {
         this._setup();
       }
       this._isAnchored = true;
+      this._onAnchorCallbacks.callCallbacks(this);
+      return this;
+    }
+
+    /**
+     * Adds a callback to be called on anchoring the Component to the DOM.
+     * If the component is already anchored, the callback is called immediately.
+     * To later deregister, use offAnchor(callback).
```

Comment:
I don't think it's worth cross-referencing the other method, since the point of the `onEvent()`/`offEvent()` naming scheme is to make things sufficiently intuitive for users.